operator/pkg/central/reconciler/reconciler_test.go

```diff
@@ -0,0 +1,46 @@
+package reconciler
+
+import (
+	"testing"
+
+	pkgReconciler "github.com/operator-framework/helm-operator-plugins/pkg/reconciler"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_addSelectorOptionIfNeeded(t *testing.T) {
```

Comment:
Can you add tests with StackRox version tags?

Reply:
I see this pattern used elsewhere, my understanding was that this pattern is used when adding tests for private (unexported) methods - that start with a lowercase character. 

- `Test_addSelectorOptionIfNeeded` same case as the method name, but uses underscore
- `TestaddSelectorOptionIfNeeded` weird
- `TestAddSelectorOptionIfNeeded` camel cased, but not same casing as method name.

Reply:
You are correct, sorry for the confusion. No need to change it.